ext/java/nokogiri/internals/c14n/UtfHelper.java

```diff
@@ -22,11 +22,10 @@
 import java.io.OutputStream;
 import java.util.Map;
 
-public final class UtfHelpper
```

Comment:
You'll also need to update the `spec.files` in `nokogiri.gemspec` if this file name has changed.